pymatviz/ptable.py

```diff
@@ -416,6 +422,10 @@ def add_child_plots(
             on_empty: Whether to "show" or "hide" tiles for elements without data.
         """
         for element in Element:
+            # Hide f-block
+            if self.hide_f_block and (element.is_lanthanoid or element.is_actinoid):
```

Comment:
could also use [`is_rare_earth_metal`](https://github.com/materialsproject/pymatgen/blob/a9da16e86f072febe3f25efe73a38f9ac611acd1/pymatgen/core/periodic_table.py#L690) here

Reply:
Thanks!

Reply:
The definition from `pymatgen` is a bit weird. By definition, the [rare-earth element](https://en.wikipedia.org/wiki/Rare-earth_element) should be Lanthanides, yttrium (Y) and Scandium (Sc)?

Reply:
i think there are [diverging opinions about this](https://www.google.com/search?q=are+actinides+rare+earth+metals&oq=are+actinides+rare). not sure `pymatgen` is the most widely held one but it would be a process to change it now

Reply:
I can see that now. But what about Y and Sc? This [doesn't seem controversial](https://www.google.com/search?q=are+Sc+and+Y+rare+earth+metals&sca_esv=e6402e358686dbdc&sxsrf=ADLYWIJsFZ1TajKcfwoF11R_eydLr4oZig%3A1715169303723&ei=F2g7ZoTnK7zvseMP1KS_8Ak&ved=0ahUKEwjEzs_I__2FAxW8d2wGHVTSD54Q4dUDCBA&uact=5&oq=are+Sc+and+Y+rare+earth+metals&gs_lp=Egxnd3Mtd2l6LXNlcnAiHmFyZSBTYyBhbmQgWSByYXJlIGVhcnRoIG1ldGFsczIIEAAYgAQYogQyCBAAGIAEGKIEMggQABiABBiiBDIIEAAYgAQYogQyCBAAGIAEGKIESLAPULYDWKQNcAJ4AZABAZgByAKgAY4MqgEHMC43LjAuMbgBA8gBAPgBAZgCBqAC2AXCAgoQABiwAxjWBBhHwgIIEAAYBxgIGB7CAggQABgFGAcYHpgDAIgGAZAGCpIHAzIuNKAHnhg&sclient=gws-wiz-serp).

Reply:
don't know how the initial implementation was motivated and it would be involved to change it now. feel free to add a PR adding the links we shared above to the [is_rare_earth_metal](https://github.com/materialsproject/pymatgen/blob/a9da16e86f072febe3f25efe73a38f9ac611acd1/pymatgen/core/periodic_table.py#L690) doc string so users are at least aware that different definitions exist

Reply:
Sure. I would do it.